test/specs/test.e2e.follow-up.js

```diff
@@ -0,0 +1,137 @@
+import { expect, browser, $ } from "@wdio/globals";
+
+describe("Follow-up claim journeys for various species", () => {
+  it("should be able to create a new follow-up claim for sheep", async () => {
+    let url = "http://localhost:3004/claim/endemics/dev-sign-in";
+
+    if (process.env.DOCKER_MODE && process.env.DOCKER_MODE === "true") {
+      url = "http://ffc-ahwr-farmer-claim:3000/claim/endemics/dev-sign-in";
+    }
+
+    await browser.url(url);
+
+    await $("#sbi").setValue("106208072");
+    await $('button[type="submit"]').click();
+
+    const sheepRadio = $('input[name="typeOfLivestock"][value="sheep"]');
+    await sheepRadio.waitForExist({ timeout: 5000 });
+    await sheepRadio.waitForDisplayed({ timeout: 5000 });
+    await sheepRadio.click();
+    await $("button=Continue").click();
+
+    await $('input[name="typeOfReview"][value="endemics"]').click();
+    await $("button=Continue").click();
+
+    const today = new Date();
+    const day = today.getDate().toString().padStart(2, "0");
+    const month = (today.getMonth() + 1).toString().padStart(2, "0");
+    const year = today.getFullYear().toString();
```

Comment:
Feels like all of this should be moved into a function, as we do it a few times.